spec/api/sul_bib/publications_api_spec.rb

```diff
@@ -192,23 +193,18 @@ def validate_authorship(pub_hash, submission)
 
       it 'creates a matching pub_hash in the publication record from the posted bibjson' do
         post_valid_json
-        pub_hash = Publication.last.reload.pub_hash
-        submission = JSON.parse(valid_json_for_post)
-        validate_authorship(pub_hash, submission)
+        validate_authorship(Publication.last.pub_hash, submission)
       end
```

Comment:
No `reload` required, huh.  Puzzled, not sure how that impacts the pub_hash content.  (Seems like maybe a case for another let to get the pub_hash content.)

Reply:
Yeah, you can watch in console when `Publication.last` hits the database.  Of course it has to check every time you call it, since other processes might have written new records.  Point being the record is freshly retrieved, so `reload` doesn't help.